colossalai/fx/passes/meta_info_prop.py

```diff
@@ -103,7 +102,7 @@ def extract_tensor_meta(obj):
             else:
                 return TensorMetadata(None, None, False, None, 0, False)
 
-        meta = map_aggregate(result, extract_tensor_meta)
+        meta = tree_map(extract_tensor_meta, result)
```

Comment:
I replace all map_aggregate with tree_map because it is not native for PyTorch development.